src/components/header/header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Header = ({ children, subheader }) =>
+  subheader
+    ? <h1 className="subheader header">{children}</h1>
+    : <h1 className="header">{children}</h1>
+
```

Comment:
@dylankfernandes It would be cleaner to use a template string here: 

className={`${subheader && "subheader"} header`}

Also, this is a good example of why I like to keep everything mapped to props. It's not wrong to split it up, but I think it's more clear if it's `props.subheader && "subheader"`

>